test/distribution/test_distribution_inference.py

```diff
@@ -347,14 +349,14 @@ def test_deterministic_l1(mu: float, hybridize: bool) -> None:
     class LaplaceFixedVarianceOutput(LaplaceOutput):
         @classmethod
         def domain_map(cls, F, mu, b):
-            b = F.ones_like(b)
-            return mu, b
+            b = 0.1 * F.ones_like(b)
+            return mu.squeeze(axis=-1), b.squeeze(axis=-1)
```

Comment:
These are also required in the `GaussianFixedVarianceOutput` a few lines above, could you add them there too?